.builder/actions/aws_crt_java_test.py

```diff
@@ -40,18 +42,19 @@ def start_maven_tests(self, env):
 
         self._run_java_tests("-DrerunFailingTestsCount=5")
 
-        # run the ShutdownTest by itself
-        env.shell.setenv('AWS_CRT_SHUTDOWN_TESTING', '1')
-        self._run_java_tests("-Dtest=ShutdownTest")
+        if os.getenv("AWS_GRAAL_VM_CI") is None:
+            # run the ShutdownTest by itself
```

Comment:
document why we don't run this with GraalVM
also: why don't we run this with GraalVM?